.travis.yml

```diff
@@ -1,15 +1,12 @@
 language: php
 
-dist: xenial
+dist: bionic
```

Comment:
do we need to define a dist at all?

Reply:
Yes, so that things don't break when travis next change their default. Though... we will probably change to github actions before long anyway, so it doesn't really matter.